server.py

```
@@ -19,240 +19,323 @@
+        'User-Agent': 'GitLabMCPCodeReview/1.0',
+        'Private-Token': gitlab_ctx.token
+    }
     
     try:
-        headers = {
-            'Accept': 'application/json',
-            'User-Agent': 'GerritReviewMCP/1.0'
-        }
-        response = requests.get(url, auth=auth, headers=headers, verify=True)
+        if method.upper() == "GET":
+            response = requests.get(url, headers=headers, verify=True)
```

Ruff (0.15.2)

[error] Probable use of `requests` call without timeout

(S113)